designer/submodules/packages/components/src/scenarios/useScenariosQuery.tsx

```diff
@@ -38,6 +38,19 @@ export function useScenariosStatusesQuery(): UseQueryResult<StatusesType> {
     });
 }
 
+export function useStatusDefinitions(): UseQueryResult<StatusDefinitionType[]> {
+    const api = useContext(NkApiContext);
+    return useQuery({
+        queryKey: ["statusDefinitions"],
+        queryFn: async () => {
+            const { data } = await api.fetchStatusDefinitions();
```

Comment:
can't be just `return await api.fetchStatusDefinitions();` or:
```
const data = await api.fetchStatusDefinitions();
return data;
```
?

Reply:
`api` methods return http response wrapped in `AxiosResponse`, here field `data` is unpacked from the object. This patter appears also in other methods.

Reply:
ehh sorry for my ignorance